tests/components/unifi/test_button.py

```diff
@@ -52,7 +55,8 @@ async def test_restart_device_button(
     # Send restart device command
     aioclient_mock.clear_requests()
     aioclient_mock.post(
-        f"https://{hub.host}:1234/api/s/{hub.site}/cmd/devmgr",
+        f"https://{config_entry.data[CONF_HOST]}:1234"
+        + f"/api/s/{config_entry.data[CONF_SITE_ID]}/cmd/devmgr",
```

Comment:
IIRC it's preferred to not use `+` for string concatenation:
```suggestion
        f"https://{config_entry.data[CONF_HOST]}:1234"
        f"/api/s/{config_entry.data[CONF_SITE_ID]}/cmd/devmgr",
```
(If that's correct, it should be changed in the other places you use this too)

Reply:
Fixed